src/System.Private.CoreLib/shared/System/Collections/Generic/Comparer.cs

```diff
@@ -82,7 +83,7 @@ public override int Compare(T x, T y)
     // Needs to be public to support binary serialization compatibility
     public sealed partial class NullableComparer<T> : Comparer<T?> where T : struct, IComparable<T>
     {
-        public override int Compare(T? x, T? y)
+        public override int Compare(T? x, T? y) // TODO-NULLABLE-GENERIC: x and y must be marked as nullable
```

Comment:
Why is this TODO-NULLABLE-GENERIC comment here?

Reply:
I don't remember why added it. It seems wrong. Will remove.